src/DataFixtures/ORM/LoadEventCategoryData.php

```diff
@@ -21,16 +21,37 @@ class LoadEventCategoryData implements FixtureInterface
         'CE010' => 'Marche',
         'CE011' => 'Support party',
     ];
-    const HIDDEN_CATEGORY_NAME = 'Catégorie masquée';
+
+    public const EVENT_CATEGORIES_GROUPED = [
+        'ancrage local' => 'event-group-category-1',
+        'projets citoyens' => 'event-group-category-1',
+        'Un An' => 'event-group-category-1',
+        'Débat' => 'event-group-category-2',
+    ];
+
+    public const HIDDEN_CATEGORY_NAME = 'Catégorie masquée';
```

Comment:
why do not include this category into the previous const array?

Reply:
add after refacto